css-view-transitions-1/Overview.bs

```diff
@@ -987,83 +1019,14 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 			Issue: Define the algorithm used to clip the snapshot when it exceeds max size.
 	</div>
 
-## [=Animate a view transition=] ## {#animate-a-view-transition-algorithm}
```

Comment:
This algorithm was being called every frame, but the things it does only need to be done once ever (like non-changing UA styles), or once at the start of each transition.

I moved the steps elsewhere.